coral/src/app/features/topics/details/subscriptions/TopicSubscriptions.tsx

```diff
@@ -123,6 +148,10 @@ const TopicSubscriptions = () => {
     });
   }, [search, teamId, aclType, selectedSubs, topicOverview]);
 
+  const selectedSub = filteredData.find(
```

Comment:
Very small comment, but I think it would be better to use a full word instead of shortening it with "sub" (would also go for "subsStats". Of course the variable names get longer, but we followed this pattern through the code (e.g. using "environment" and not "env") and it would be good trying to keep it consistent. Also helps with reading the code, since there's no moment where you have to translate the shortener to the real word in your brain.

Reply:
Very good point, I actually thought the same myself at some point but then got too into it and blind to the issue ^^ Will fix